beacon-chain/rpc/beacon_chain_server.go

```diff
@@ -72,3 +73,53 @@ func (bs *BeaconChainServer) ListValidatorBalances(
 	}
 	return &ethpb.ValidatorBalances{Balances: res}, nil
 }
+
+// GetValidators retrieves the active validators with an optional historical epoch flag to
+// to retrieve validator set in time.
+//
+// TODO(#3045): Implement validator set for a specific epoch. Current implementation returns latest set,
+// this is blocked by DB refactor.
+func (bs *BeaconChainServer) GetValidators(
+	ctx context.Context,
+	req *ethpb.GetValidatorsRequest) (*ethpb.Validators, error) {
+
+	validators, err := bs.beaconDB.Validators(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not retrieve validators: %v", err)
+	}
+
+	// Return the entire validator list if option PageToken is not specified.
+	if req.PageToken == "" {
+		return &ethpb.Validators{
+			Validators:    validators,
```

Comment:
About about page size? This would return all of the validators in the database?

Reply:
Yes, without `PageToken` all the validators will be returned. Did you want to add some constraints with the `PageSize`?

Reply:
Fixed